cmd/manager/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/kudobuilder/kudo/pkg/controller/operator"
 	"github.com/kudobuilder/kudo/pkg/controller/operatorversion"
 	"github.com/kudobuilder/kudo/pkg/version"
+	apiextenstionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Typo? `extenstions`

Reply:
phew at least it's consistent! :) will get that fixed.